docs/site/Working-with-data.md

```diff
@@ -0,0 +1,89 @@
+---
+lang: en
+title: 'Working with data'
+keywords: LoopBack 4.0, LoopBack 4
+layout: readme
+source: loopback-next
+file: packages/metadata/README.md
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Working-with-data.html
+summary: --
+---
+
+In LoopBack 4, models describe the shape of data, repositories provide behavior
+like CRUD operations, and controllers define routes (this is different from
+LoopBack 3.x where models implement behavior too). LB4
+[repositories](Repository.md) provide a couple of create, read, update, and
```

Comment:
do the CRUD operations only available for the CRUDRepository?

Reply:
I think for now all LB4 repositories are extended from `DefaultCrudRepository`, and it provides those CRUD operations mentioned here. If this is true, then I think the answer is yes.

Reply:
cuz i'm thinking Repository doesn't need to have CRUD, so that's why we might want to leave it for a broader usage.